app/src/main/java/fr/neamar/kiss/result/Result.java

```diff
@@ -404,6 +410,7 @@ public long getUniqueId() {
             this.appResultWeakReference = new WeakReference<>(result);
         }
 
+        @SuppressLint("WrongThread")
```

Comment:
This doesn't seem like a good idea.

Reply:
What would you change?

Reply:
I would remove the annotation, so that warning keeps showing and can be fixed later.